### PR TITLE
Schedule breakpoint fixed

### PR DESCRIPTION
**Pull request by realslimshanky**

### CODE REVIEW

**Comment by akshayaurora** [comment]
@realslimshanky thanks

**Comment by realslimshanky** [comment]
anytime :)

File: js/main.js [hunk 1]
```diff
@@ -5237,10 +5237,10 @@ function insertTableRows(table, rows) {
 	$(rows).each(function() {
         var row = $(this);
         row_html += '<tr>' +
-                       '<td>' + row[0] +'</td>' +
-                       '<td>' + row[1] +'</td>' +
-                       '<td>' + row[2] +'</td>' +
-                       '<td>' + row[3] +'</td>' +
+                       '<td class="first_col">' + row[0] +'</td>' +
```
Comment (aktech) [inline js/main.js:5240]: This would not work.

Comment (realslimshanky) [reply js/main.js:5240]: There seems to be no problem on the website because of it.

**Pull request merged**